confgenerator/feature_tracking_test.go

```diff
@@ -464,10 +464,10 @@ func TestOverrideDefaultPipeline(t *testing.T) {
 
 func TestGolden(t *testing.T) {
 	_ = apps.BuiltInConfStructs
-	components := confgenerator.GetComponentsFromRegistry(confgenerator.LoggingReceiverTypes)
-	components = append(components, confgenerator.GetComponentsFromRegistry(confgenerator.LoggingProcessorTypes)...)
-	components = append(components, confgenerator.GetComponentsFromRegistry(confgenerator.MetricsReceiverTypes)...)
-	components = append(components, confgenerator.GetComponentsFromRegistry(confgenerator.MetricsProcessorTypes)...)
+	components := confgenerator.LoggingReceiverTypes.GetComponentsFromRegistry()
+	components = append(components, confgenerator.LoggingProcessorTypes.GetComponentsFromRegistry()...)
+	components = append(components, confgenerator.MetricsReceiverTypes.GetComponentsFromRegistry()...)
+	components = append(components, confgenerator.MetricsProcessorTypes.GetComponentsFromRegistry()...)
```

Comment:
We were going to leave it for later since it doesn't need to block this PR, but if it's just a one-liner then I may as well add it in. Thanks for pointing this out!

Reply:
Actually upon second glance this looks like a non-trivial change, so I'd prefer to leave it for a future PR.